src/AppBundle/Entity/Post.php

```diff
@@ -53,6 +53,7 @@ class Post
 
     /**
      * @ORM\Column(type="text")
+     * @Assert\NotBlank(message="post.blank_content")
```

Comment:
You are right. That will prevent us from having to update all translations. I've done that in https://github.com/symfony/symfony-demo/commit/d68caa051ae7717b4ddb0e61650fa78bad226ab9 to merge this contribution "as is". Thanks!